src/js/actions/sampler.js

```diff
@@ -211,16 +211,12 @@ define(function (require, exports) {
      * @return {Promise}
      */
     var applyColor = function (doc, layers, color) {
```

Comment:
so what's the point of passing layers to this now, if we're just gonna ignore it?